src/inject/dynamic-theme/style-manager.ts

```diff
@@ -305,7 +305,7 @@ export function manageStyle(element: StyleElement, {update, loadingStart, loadin
                 isAsyncCancelled: () => cancelAsyncOperations,
             });
             isOverrideEmpty = syncStyle.sheet.cssRules.length === 0;
-            if (sheetModifier.hasNotLoadedImports()) {
+            if (sheetModifier.getNotLoadedLinkStatus() === 'found') {
```

Comment:
It would be a bit better if Style Manager didn't know about statuses, but just knew if it should rebuild the style, or not. But at the same time you can just make sure this update is executed only once. Just add some lock to the Style Manager, e.g. a flag telling it has already reacted on error.